### PR TITLE
Pipelines description added

### PR DESCRIPTION
**Pull request by TiffanyLuce**
You can now add a description to a pipeline and if there's one it will show when you hover over the pipeline name

### CODE REVIEW

**Comment by adambrown1** [comment]
The functionality is great!
It would be even nicer if the design could be the same as the tip you get hovering over the stop, restart, silence buttons -- something like the data-bs-toggle option I think.

**Comment by adambrown1** [comment]
Partly solves #16

**Comment by TiffanyLuce** [comment]
Design updated ! Better ?
I guess it's nice if we avoid really long descriptions

**Review by adambrown1 — COMMENTED** [review]
I like it. Have one comment, not really part of your changes but related. I leave it to Jaron to approve PRs on his baby ;)

File: public/javascripts/pipeline.js [hunk 1]
```diff
@@ -30,7 +30,8 @@ function PopulatePipelines(flavor) {
       var n = doc.name;
       if (n.toUpperCase().indexOf(filter) > -1) {
         if (doc.status == 'active') {
-          var row = `<tr><td onclick="PipelineDropdown('${n}')">${n}</td>`;
+          if (doc.description == undefined) var row = `<tr><td onclick="PipelineDropdown('${n}')">${n}</td>`;
+          else var row = `<tr><td data-bs-toggle="tooltip" title="${doc.description}" onclick="PipelineDropdown('${n}')">${n}</td>`;
           try{
             row += `<td>${doc.rate.toPrecision(3)}</td> <td>${(doc.dt || 0).toPrecision(1)}</td> <td>${doc.cycle-doc.error}</td>`;
```
[the remark this answers is not in the document]
Comment (TiffanyLuce) [reply public/javascripts/pipeline.js:36]: Oh so initializing the 'row' variable and checking for the description to be there before the if (line 32) and then incrementing it as a function of the pipeline status (inactive...) ?

[the remark this answers is not in the document]
Comment (adambrown1) [reply public/javascripts/pipeline.js:36]: That's what I meant. Just avoids having 3x identical code.

**Review by jarongrigat — APPROVED** [review]

**Pull request merged**